src/Pages/SingleVideoPage/SingleVidePageComponents/VideoDescription/VideoDescription.jsx

```diff
@@ -0,0 +1,30 @@
+import { useParams } from "react-router-dom";
+import { useReducerContext } from "../../../../Context";
+import "./VideoDescription.css";
+
+export const VideoDescription = () => {
+  const { videoId } = useParams();
+  const { videos } = useReducerContext();
+
+  return videos
+    .filter((video) => video._id === videoId)
+    .map((video) => {
+      return (
```

Comment:
Suggestion : You can try destructuring values in video, so that you can straight away use them instead of using `video.attribute_name` everytime. Not necessary tho.

Reply:
Yeah thanks for pointing it out. I will change it